docs/built-in-functions.rst

```diff
@@ -48,14 +48,28 @@ contracts.
     * ``value``: Value for the ether unit
     * ``unit``: Ether unit name (e.g. ``"wei"``, ``"ether"``, ``"gwei"``, etc.) indicating the denomination of ``value``.
 
-.. py:function:: slice(b: bytes, start: int128, length: int128) -> bytes
+.. py:function:: slice(b: Union[bytes, bytes32, string], start: uint256, length: uint256) -> Union[bytes, string]
```

Comment:
Does it make sense to be able to do a slice of a `bytes32`?

Reply:
Yeah, I've personally used it when rolling my own tight storage packing logic.

Reply:
ehhhhhh....